storage/perfschema/my_thread.h

```diff
@@ -10,6 +10,10 @@
 #include <sys/syscall.h>
 #endif
 
+#ifdef HAVE_PTHREAD_GETTHREADID_NP
+#include <pthread_np.h>
+#endif
+
```

Comment:
I would also suggest to change line `76` comment to `HAVE_PTHREAD_THREADID_NP` instead of `HAVE_SYS_THREAD_SELFID `.

Reply:
the condition before line `76` covers `defined(HAVE_PTHREAD_GETTHREADID_NP)` and line `76` is in an an `#else` following this condition.